runtime/src/lib.rs

```diff
@@ -143,15 +143,20 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
     transaction_version: 1,
 };
 
-pub const MILLISECS_PER_BLOCK: u64 = 6000;
```

Comment:
could this all be moved into pallets/primitives/src/constants/mod.rs, like in standalone chain, and possible PR to improve substrate-parachain-template?

Reply:
Yes. we will do it as a part of https://github.com/DataHighway-DHX/node/issues/230